### PR TITLE
fix(cli): throw error with proper code for failed imports

### PR DESCRIPTION
**Pull request by pi0**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

The usage of `_import` was originally expanding search path. Using `consola.fatal` disallows catching errors.

This PR fixes problem with typescript package detection by not forcing the user to install `@nuxt/typescript` when `tsconfig.json` exists in a project. ([source](https://github.com/nuxt/nuxt.js/blob/v2.6.1/packages/cli/src/utils/typescript.js#L22)) -- Thanks to @rchl for reporting this issue.

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/cli/src/imports.js [hunk 1]
```diff
@@ -5,24 +5,22 @@ const localNodeModules = path.resolve(process.cwd(), 'node_modules')
 
 // Prefer importing modules from local node_modules (for NPX and global bin)
 async function _import(modulePath) {
-  let m
-  for (const mp of [ path.resolve(localNodeModules, modulePath), modulePath ]) {
+  for (const mp of [
+    path.resolve(localNodeModules, modulePath),
+    modulePath
+  ]) {
     try {
```
Comment (clarkdo) [inline packages/cli/src/imports.js]: Is await for more readable?

Comment (pi0) [reply packages/cli/src/imports.js]: Without await, (and returning promise) we cannot catch error here. One way would be separating it like this:

```js
const m = await import(mp)
return m
} catch(e) [
```

But I taught one-liner would be more readable.

Comment (clarkdo) [reply packages/cli/src/imports.js]: Will throw inside catch instead of end be better?

Comment (pi0) [reply packages/cli/src/imports.js]: We have to only throw an error if all possible locations (for loop) has been checked. Throwing inside the catch of for loop requires one additional check (`m === mp`) to see if we are reaching to the end or not. This is same as refactoring out of for loop and using early return

Comment (clarkdo) [reply packages/cli/src/imports.js]: Gotcha

**Review by clarkdo — DISMISSED** [review]

**Pull request merged**